iceberg/iceberg-handler/src/test/queries/positive/iceberg_major_compaction_partition_evolution_ordered.q

```diff
@@ -63,7 +64,7 @@ describe formatted ice_orc;
 explain alter table ice_orc COMPACT 'major' and wait order by first_name desc;
 explain optimize table ice_orc rewrite data order by first_name desc;
 
-alter table ice_orc COMPACT 'major' and wait order by first_name desc;
+optimize table ice_orc rewrite data order by first_name desc;
```

Comment:
Why are we removing tests with the old HiveQL? 
is `optimize` table always blocking?

Reply:
Fixed